runtime/src/bank/check_transactions.rs

```diff
@@ -126,6 +146,28 @@ impl Bank {
             .collect()
     }
 
+    fn checked_transactions_details_with_test_override(
+        nonce: Option<NonceInfo>,
+        lamports_per_signature: u64,
+        compute_budget_and_limits: Result<
+            SVMTransactionExecutionAndFeeBudgetLimits,
+            TransactionError,
+        >,
+    ) -> CheckedTransactionDetails {
+        let compute_budget_and_limits = if lamports_per_signature == 0 {
+            // This is done to support legacy tests. The tests should be updated, and check
```

Comment:
This is essentially a leftover `TODO`, right?

Reply:
Yes, this check is moved out of SVM, so that it can be handled in monorepo.